src/main/scala/workflow/graph/Estimator.scala

```diff
@@ -0,0 +1,63 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+/**
+ * An estimator has a `fitRDD` method which takes an input and emits a [[Transformer]]
```

Comment:
So this looks like another change to the programming model? E.g. all Estimators now have to support a fitRDD method? Why can't they just support a `fit` method?